es-vue-base/src/lib-components/EsZipCodeForm.vue

```diff
@@ -86,9 +85,7 @@
                         class="text-nowrap"
                         :class="dark ? 'text-white' : ''"
                         :target="privacyPolicyNewTab ? '_blank' : '_self'">
-                        <slot name="privacyPolicyLinkText">
-                            Privacy Policy
-                        </slot>
+                        <slot name="privacyPolicyLinkText"> Privacy Policy </slot>
```

Comment:
We should remove extraneous stylistic changes, especially since it causes this warning